C-set/1740C-Bricks-and-bags.cpp

```diff
@@ -49,177 +37,54 @@ typedef map<ll,ll> mll;
 #define all(v) v.begin(), v.end()
 #define rall(v) v.rbegin(), v.rend()
 #define sz size()
-//---------------------Global Constants and Functions-----------------------//
 
- 
-//-----------------------------Code begins----------------------------------//
+const int maxn = 2 * 1e5 + 5;
 
-const int maxn = 2*1e5 + 5;
+void solve() {
 
- void solve(){
-   
-   ll n;
-   cin >> n;
-   vi a(n);
-   cin >> a;
+  ll n;
+  cin >> n;
+  vi a(n);
+  cin >> a;
   sort(all(a));
-   set<ll> k;
-   rep(i,n)k.insert(a[i]);
-   if(k.size() == 1){
+  set < ll > k;
+  rep(i, n) k.insert(a[i]);
+  if (k.size() == 1) {
     cout << 0 << '\n';
     return;
-   }
- 
- 
-   ll ans = -1;
-  
-   for(int i =0;i<n;i++){
-      if(i == 0){
-         ans = max(ans, a[n-1] + a[1] - 2*a[0]);
-      }
-      elif(i == n-1){
-         ans = max(ans , 2*a[n-1] - a[0] - a[n-2]);
-      }
-      else{
-         ans = max(ans, max(a[i+1]-a[i],a[i]-a[i-1]) + max(a[n-1]-a[i],a[i]-a[0]));
-      }
-
-   } 
-
-   cout << ans <<'\n';
-
-
-   
+  }
 
+  ll ans = -1;
 
+  for (int i = 0; i < n; i++) {
+    if (i == 0) {
+      ans = max(ans, a[n - 1] + a[1] - 2 * a[0]);
+    }
+    elif(i == n - 1) {
```

Comment:
this should be `else if`

Reply:
These things have already been mentioned in the defines @Waqar-107